sources/api/netdog/src/net_config/devices/mod.rs

```diff
@@ -0,0 +1,118 @@
+//! The devices module contains all the types of network devices that `netdog` supports. These are
```

Comment:
I'm still learning, so sorry if this is a stupid question. Isn't `//!` used for inner block comments?